### PR TITLE
jest option to bypass modular behaviour  

### PR DESCRIPTION
**Pull request by AlbertoBrusa**
- [x] Add flag to `modular test` which sends all flags directly to Jest 
- [x] Add documentation with instructions to configure IntelliJ for modular-test
- [ ] ~~Fix jest logs not appearing in IntelliJ~~

### CODE REVIEW

File: docs/commands/test.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ empty, Modular will write a message to `stdout` and exit with code `0`.
 
 ### Modular-specific options
 
+`--jest`: Bypass Modular selective behaviour and flags, and send all provided
```
Comment (cristiano-belloni) [inline docs/commands/test.md:62]: My proposals:
- `--bypass`
- `--bypassToJest`

Comment (AlbertoBrusa) [reply docs/commands/test.md:62]: bypass sounds good to me me

**Comment by cristiano-belloni** [comment]
@AlbertoBrusa LOL, what are the v99.0.0, v999.0.0, v9999.0.0 etc. commits?

**Review by cristiano-belloni — APPROVED** [review]

**Comment by cristiano-belloni** [comment]
Code seems good, but it looks like one test is not passing

**Comment by AlbertoBrusa** [comment]
> @AlbertoBrusa LOL, what are the v99.0.0, v999.0.0, v9999.0.0 etc. commits?

Verdaccio - I was testing it with verdaccio and used the wrong flag when publishing, so it automatically commits version changes

**Pull request merged**